src/main/java/uk/gov/hmcts/ccd/security/filters/ServiceAuthFilter.java

```diff
@@ -0,0 +1,73 @@
+package uk.gov.hmcts.ccd.security.filters;
+
+import org.slf4j.Logger;
```

Comment:
glad to see that this has now been included in https://github.com/hmcts/service-auth-provider-java-client/pull/46.
I guess now we are going to remove this and take it from `service-auth-provider-java-client` right?